src/actions/authActions.js

```diff
@@ -1,11 +1,13 @@
-/* globals window, self */
+/* globals window, self, Promise */
 
 // auth0 actions
 // 3 exported functions:
 // - start()
 // - login()
 // - logout()
 
+import get from 'lodash.get'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'lodash.get'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->